lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="pagination-wrapper" [ngClass]="className" id="id">
+    <nav class="custom-pagination">
+        <ul class="pagination">
+            <ng-container *ngIf="value !== 1 && useFirstAndLast">
+                <li class="page-item" (click)="handleOnChange(1)">
+                    <button class="page-link" title="firstText">
```

Comment:
Don't forget the binding
```suggestion
                    <button class="page-link" [title]="firstText">
```